src/components/ButtonPanel.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Button from "./Button";
+export default class ButtonPanel extends React.Component {
+  render() {
+    return (
+      <div>
+        <div class="buttons">
```

Comment:
If possible let us eliminate the duplication of `Button` here by using array `map`